zulip_bots/zulip_bots/lib_tests.py

```diff
@@ -138,3 +143,36 @@ def test_message(message, flags):
                                         config_file=None,
                                         bot_config_file=None,
                                         bot_name='testbot')
+
```

Comment:
I would consider refactoring the following two test functions to use a helper, given the duplication.

Reply:
Add `_create_client_and_handler_for_file_upload()` helper method